aptible/resource_endpoint_test.go

```diff
@@ -546,6 +574,45 @@ func testAccAptibleEndpointUpdateIPWhitelist(appHandle string) string {
 	return output
 }
 
+func testAccAptibleEndpointBadPort(appHandle string) string {
+	// Use a bad port to make the provision operation fail
+	output := fmt.Sprintf(`
+	resource "aptible_environment" "test" {
+		handle = "%s"
+		org_id = "%s"
+		stack_id = "%v"
+	}
+
+	resource "aptible_app" "test" {
+		env_id = aptible_environment.test.env_id
+		handle = "%v"
+		config = {
+			"APTIBLE_DOCKER_IMAGE" = "nginx"
+		}
+		service {
+			process_type = "cmd"
+			container_memory_limit = 512
+			container_count = 1
+		}
+	}
+
+	resource "aptible_endpoint" "test" {
+		env_id = aptible_environment.test.env_id
+		resource_id = aptible_app.test.app_id
+		resource_type = "app"
+		process_type = "cmd"
+		endpoint_type = "https"
+		managed = true
+		domain = "www.aptible-test-demo.fake"
+		internal = true
+		platform = "alb"
+		container_port = 666
+	}
+`, appHandle, testOrganizationId, testStackId, appHandle)
+	log.Println("HCL generated: ", output)
```

Comment:
println that you probably meant to remove here

Reply:
We print the HCL for all the tests in this file to make it easier to debug.